client/components/checklist/checklist.js

```diff
@@ -129,8 +135,23 @@ export default class Checklist extends PureComponent {
 							onTaskClick: () => this.setExpandedTask( realIndex ),
 						} );
 					} ) }
+
+					{ wpAdminUrl && showAdminFooter && (
+						<JetpackChecklistFooter
```

Comment:
Feels too funky to have Jetpack specific thing inside this generic checklist block. :-)

Let's either rename it to just `ChecklistFooter` provide its contents via prop or provide the whole component as a prop (preferable since it's pretty custom).

Note that this move dropped out the `handleWpAdminLink` handler.

Reply:
> provide the whole component as a prop (preferable since it's pretty custom).

Sounds like a good way to go. I'll update it accordingly. 

> Note that this move dropped out the handleWpAdminLink handler that sends tracks events.
Shoot. Thanks for catching that. The new approach should handle that better.

Reply:
You can drop this in browser console to see tracks events live as you click on things. Makes debugging these a breeze. :-)
```
localStorage.debug = 'calypso:analytics:tracks';
```